src/ddmd/traits.d

```diff
@@ -403,6 +404,47 @@ extern (C++) Expression semanticTraits(TraitsExp e, Scope* sc)
         return new IntegerExp(e.loc, false, Type.tbool);
     }
 
+    /**
+     if traitsFuncArg is passed either a delegate. a function
+     or a function pointer or a variable of types.
+     it will return the functionType.
+     else it will return null
+
+     In case of a function declaration being passed verbatim
+     it will set the second  optional argument to the FuncDeclarartion
+     given it was provided.
+    */
```

Comment:
@UplinkCoder don't forget that we already publish the documentation of ddmd on dlang.org:
- http://dlang.org/phobos-prerelease/ddmd_mars.html (Ddoc)
- http://dlang.org/library-prerelease/ddmd/mars.html (Ddox)
and we intend to publish it on the main page very soon (e.g. once #6771 is merged).

This is also why it can be very helpful for both the contributor and the reviewers to preview the style changes at Vladimir's excellent DAutoTest, e.g.

http://dtest.dlang.io/artifact/website-72ba79b73f635fb08bdfe69d6826833c1c4c2ee0-6f020744a26d63533038dc39e44ae752/web/library-prerelease/ddmd/traits.html

(in this case it's a function within a function and thus _not_ publicly accessible)